modules/eco-poison-pill-operator-installation-web-console.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="installing-poison-pill-operator-using-web-console_{context}"]
+= Installing the Poison Pill Operator using the web console
+
+The following procedure installs the Poison Pill Operator using the {product-title} web console.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware using installer-provisioned infrastructure (IPI).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
```

Comment:
We should mention something about modifying the MHC CR to trigger Poison Pill. not sure where's the right place for that

Reply:
Added a new section (procedure) to address this